docs/data/charts/bar-demo/PopulationPyramidBarChart.js

```diff
@@ -38,10 +39,30 @@ const female = [
   168603, 36739, 5770,
 ];
 
-const numberFormatter = Intl.NumberFormat(undefined, {
+const numberFormatter = Intl.NumberFormat('en-US', {
   useGrouping: true,
 });
 
+function formatWithSuffix(number) {
+  const suffixes = [
+    { value: 1e6, symbol: 'M' },
+    { value: 1e3, symbol: 'k' },
+  ];
+
+  const suffix = suffixes.find((s) => Math.abs(number) >= s.value);
+
+  if (!suffix) {
+    return new Intl.NumberFormat('en-US').format(number);
+  }
+
+  const formattedValue = new Intl.NumberFormat('en-US', {
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 1,
+  }).format(number / suffix.value);
+
+  return formattedValue + suffix.symbol;
```

Comment:
```suggestion
  return new Intl.NumberFormat('en-US', { notation: 'compact' }).format(number);
```

![image](https://github.com/user-attachments/assets/5d9e2007-c42f-450b-9897-4dfaa15de7ac)

Reply:
Much better, thank you 😄